main.py

```diff
@@ -1,7 +1,37 @@
-import init_django_orm  # noqa: F401
-
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre_name in genres:
+        Genre.objects.get_or_create(name=genre_name)
+
+    # Create actors
+    actors = [
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    ]
+    for first_name, last_name in actors:
+        Actor.objects.get_or_create(first_name=first_name, last_name=last_name)
```

Comment:
Please add some blank lines between create, update and delete actions code blocks